server/portal_server.js

```diff
@@ -4,12 +4,24 @@ const path = require('path');
 const fs = require('fs-extra');
 const bodyParser = require('body-parser');
 const crypto = require('crypto');
+const conf = require('./config.js');
+const stream = require('stream');
+const fetch = require('node-fetch');
+const storage = require('./storage.js');
+
+let isProduction =
```

Comment:
Nit: I don't think this variable is used in this file anymore.
But @dannycoates comment still applies to server/storage.js:10 below.